Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -102,14 +99,14 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintCallable, Category = "C++")
 	void UnlockAllLevels();
 
-	/** Is called to update the stars actors and in widgets */
-	UFUNCTION(BlueprintCallable, Category = "C++")
-	void RefreshProgressionUIElements();
-
 	/** Returns difficultyMultiplier */
-	UFUNCTION(BlueprintCallable, Category="C++")
+	UFUNCTION(BlueprintCallable, BlueprintPure, Category="C++")
```

Comment:
All Getters functions have to be `const` at the end: `float GetDifficultyMultiplier() const;`

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/a355424c4ff3b17ded32dc2185e8fa2823695913